src/Pages/Auth/Register/index.js

```diff
@@ -1,15 +1,113 @@
-import React from "react"
+import React, { useState } from "react"
+import { useQuery } from "react-query"
 import { Link } from "react-router-dom"
+import Input from "components/Input"
+import Button from "components/Button"
+import { useI18n } from "providers/I18n"
 
 Register.propTypes = {}
 
 function Register() {
+  const { t } = useI18n()
+
+  const [urlUserCode, setUrlUserCode] = useState(window.location.href)
+
+  const [formData, setFormData] = useState({
+    email: "",
+    password: "",
+    firstName: "",
+    lastName: "",
+    dob: "",
+    profession: "",
+    passwordConfirmed: ""
+  })
+
+  const getUserData = async () => {
+    const response = await fetch(`https://randomuser.me/api/`)
+    const data = await response.json()
+    return data
```

Comment:
This has the same effect
```suggestion
    return response.json()
```